app/assets/javascripts/dashboard.coffee

```diff
@@ -25,6 +26,24 @@ synchronizeCourse = (event) ->
       $("div.user-courses-container").html(data.partial)
   event.preventDefault()
 
+synchronizeDatesDashboard = (event) ->
+  url = "/user_dates/synchronize_dates_on_dashboard.json"
+  $.ajax
+    url: url
+    method: 'GET'
+    error: (jqXHR, textStatus, errorThrown) ->
+      console.log('error_synchronize_dates')
+      alert(I18n.t('global.ajax_failed'))
+    success: (data, textStatus, jqXHR) ->
+      console.log('success_synchronize_dates')
```

Comment:
I don't remember our policy for log messages in a case of a successful ajax. As far as I can see it within this file, there are only log messages in case of any error.